src/Schema/Directives/LikeDirective.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Nuwave\Lighthouse\Schema\Directives;
+
+use Nuwave\Lighthouse\Support\Contracts\ArgBuilderDirective;
+
+class LikeDirective extends BaseDirective implements ArgBuilderDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+enum LikePercentageLocation {
+    START
+    END
+    BOTH
+}
+
+"""
+Use the client given value to add a `LIKE` conditional to a database query.
+"""
+directive @like(
+  """
+  Specify the database column to compare.
+  Only required if database column has a different name than the attribute in your schema.
+  """
+  key: String
+
+  """
+  Specify the positions of the % in the LIKE comparison. The default is BOTH.
+  """
+  percentage: LikePercentageLocation
+) repeatable on ARGUMENT_DEFINITION | INPUT_FIELD_DEFINITION
+GRAPHQL;
+    }
+
+    protected function escapePercentage(string $value): string
+    {
+        $char = '\\';
```

Comment:
That's a variable. Given the value never changes, we should just use an actual `const`.

Reply:
fixed